src/test/java/org/radarbase/appserver/controller/FcmNotificationControllerTest.java

```diff
@@ -113,7 +114,7 @@ public void setUp() {
             .setDelivered(false)
             .setId(2L);
 
-    given(notificationService.addNotification(notificationDto2, USER_ID, PROJECT_ID))
+    given(notificationService.addNotification(notificationDto2, USER_ID, PROJECT_ID, SCHEDULE_NOW))
```

Comment:
You can leave this as it is and add additional tests for the schedule functionality. This would ensure that both cases are checked (when `schedule` is true and when false). This would also mean we checked that this is backward compatible.